### PR TITLE
Sorted through remaining PartMC input generation issues

### PR DESCRIPTION
**Pull request by jeff-cohere**
This branch was named after the postprocessing analysis I want to be doing, but it turns out we had to work through some PartMC input issues first. Let's get these merged.

### CODE REVIEW

File: ambrs/aerosol.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ class AerosolModeDistribution:
     species: tuple[AerosolSpecies, ...]
     number: RVFrozenDistribution                     # modal number concentration distribution
     geom_mean_diam: RVFrozenDistribution             # geometric mean diameter distribution
+    log10_geom_std_dev: float                        # mode-specific logarithmic diameter std dev
```
Comment (lfierce3) [inline ambrs/aerosol.py:70]: looks good! later, this could be something we sample (e.g., RVFrozenDistribution type)

File: demo.py [hunk 2]
```diff
@@ -94,6 +92,7 @@
                 species = [so4, pom, soa, bc, dst, ncl],
                 number = stats.loguniform(3e7, 2e12),
                 geom_mean_diam = stats.loguniform(0.5e-7, 1.1e-7),
+                log10_geom_std_dev = 1.6,
```
Comment (lfierce3) [inline demo.py:95]: This isn't quite right. geom_std_dev=1.6, so log10_geom_st_dev = np.log10(1.6)

Comment (jeff-cohere) [reply demo.py:95]: Oh, right. I can fix all of those. Been running around a bit too much.

**Review by lfierce3 — CHANGES_REQUESTED** [review]
See comments on log10_geom_std_dev. Can't remember which file it's in.

File: demo.py (hunk 2 above)
Comment (lfierce3) [reply demo.py:95]: I'm not sure if it's bad form for me to fix it myself or not! I was just looking for it. :)

Comment (jeff-cohere) [reply demo.py]: I think there's a way to suggest a fix, which can be committed right from here. We should sit down and work through a code review when it's convenient. (I have a meeting in a few minutes, so I just wanted to get these in!)

**Pull request merged**